### ISSUE TITLE
iOS 13 issue and work around

### ISSUE THREAD

**Issue by lewis-smith** [sobri909/MGMushParser#7]
In iOS 13 I found that:

`MGMushParser attributedStringFromMush:description font:font color:color];`

would use a sans serif font for any bold text. I think this is down to changes in how to instantiate system fonts in iOS 13.

Of course it would be great if this was fixed, but in the mean time you can work around this by specifying bold fonts by hand:

`[MGMushParser attributedStringFromMush:description
                                                    font:font
                                                boldFont:boldFont
                                              italicFont:italicFont
                                                   color:color
                                           paragraphStyle:paragraphStyle];`

**Comment by saiday** [comment]
Yep, other convenient constructors should provide system bold and italic font instead of nil font to parser.

I can submit a PR if the statement is doable.
